app/src/main/java/it/quip/android/partials/EndlessScrollHandler.java

```diff
@@ -0,0 +1,47 @@
+package it.quip.android.partials;
```

Comment:
Hasham had me move it from that exact pattern

On Mon, Oct 19, 2015 at 3:40 PM, Jonathan Como <notifications@github.com>
wrote:

> In app/src/main/java/it/quip/android/partials/EndlessScrollHandler.java
> <https://github.com/quipit/quipit-android/pull/9#discussion_r42435390>:
>
> > @@ -0,0 +1,47 @@
> > +package it.quip.android.partials;
>
> "Interfaces" shouldn't be a package name. The interfaces should lie within
> the appropriate packages. That would be like making a "classes" or "enums"
> package
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quipit/quipit-android/pull/9/files#r42435390>.
>